tpm2/policy.go

```diff
@@ -20,420 +20,255 @@
 package tpm2
```

Comment:
the diff of this file is too messy to follow, there's too much renaming/removing and moving going on at the same time, maybe it's hard to avoid but  my main suggestion would be to try to avoid some of  the moving and do a separate PR that fixes the order of things, examples, we probably want the new interfaces at the top and then

*  pcrPolicyParams close to where dynamicPolicyComputeParams was
* policyOrNode close to where policyOrDataNode was
* policyOrTree close to where policyOrDataTree was
* the errors closer to where the previous error def was
* ensureSufficientORDigests where it was before
* pcrPolicyCounterContext  maybe where pcrPolicyCounterHandle was
* (t *policyOrTree) executeAssertions close to where executePolicyORAssertions was ?
* newPolicyOrTree close to where computeDynamicPolicy was ?

etc, maybe it's impossible let me know?

Reply:
Thanks, I've moved things around a bit. It has reduced the diff slightly (+5,031 −3,291 -> +5,009 −3,269), although I'm not sure if it's made it much more readable.